airflow/api_internal/endpoints/rpc_api_endpoint.py

```diff
@@ -33,11 +32,17 @@ def _build_methods_map(list) -> dict:
     return {f"{func.__module__}.{func.__name__}": func for func in list}
 
 
-METHODS_MAP = _build_methods_map(
-    [
-        DagFileProcessor.update_import_errors,
-    ]
-)
+def _initialize_map() -> dict:
```

Comment:
I think we should inline `_build_methods_map` into this.

Reply:
Sure.